input/examples/location-patient-home.xml

```diff
@@ -6,7 +6,6 @@
     </meta>
     <status value="active"/>
     <name value="Patient's home"/> 
-    <mode value="kind"/>
```

Comment:
Restore Location.mode. No it isn't Must Support but is critical for a Patient Home example.